appendices/appendix_a_piano_rolls.md

```diff
@@ -0,0 +1,3 @@
+# Piano Rolls
```

Comment:
I am thinking appendix A should be a listing of site swaps for each pattern in the book.  Something that readers can input into the Juggling Lab easily.

I think this might be better as appendix f, I had made it appendix a so that Graham's would be appendix g .  B^D